### DISCUSSION TITLE
ReSpeaker 6-Mic Circular Array Kit for Raspberry Pi | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#630] [Q&A]
# ReSpeaker 6-Mic Circular Array Kit for Raspberry Pi | Seeed Studio Wiki

ReSpeaker 6-Mic Circular Array Kit for Raspberry Pi

https://wiki.seeedstudio.com/ReSpeaker_6-Mic_Circular_Array_kit_for_Raspberry_Pi/

**Comment by vmanoj1996** [comment]
can it return the raw voice output for 6 channels at 48 KHz. We want to write the audio processing algorithms ourselves

**Comment by bengizmo** [comment]
Is this compatible with the Raspberry Pi 5?
